packages/opentelemetry-sdk-trace-base/test/common/export/BatchSpanProcessorBase.test.ts

```diff
@@ -175,43 +175,35 @@ describe('BatchSpanProcessorBase', () => {
       assert.strictEqual(spy.args.length, 0);
     });
 
-    it('should export the sampled spans with buffer size reached', done => {
-      const clock = sinon.useFakeTimers();
+    it('should export the sampled spans with buffer size reached', async () => {
```

Comment:
Looks like this test is now modified to test the eager behavior. not sure what it was testing before since the name isn't very good.